src/services/web3/web3.plugin.ts

```diff
@@ -66,17 +67,20 @@ type PluginState = {
   walletState: WalletState;
 };
 
-async function isSanctionedAddress(address: string): Promise<boolean | null> {
-  try {
-    const response = await axios.post(SANCTIONS_ENDPOINT, [
-      {
-        address: address.toLowerCase(),
-      },
-    ]);
-    const isSanctioned = response.data[0].isSanctioned;
-    return isSanctioned;
-  } catch {
-    return null;
+async function isSanctionedAddress(address: string): Promise<boolean> {
+  if (configService.env.WALLET_SCREENING) {
```

Comment:
Try and avoid else statements as they're almost never needed. This can be replaced with `if (!configService.env.WALLET_SCREENING) return false;`

Reply:
It was specifically requested to be like this by @garethfuller

Reply:
Yea he was just saying to return a boolean instead of null. I'm saying we can clarify the code by removing the else portion and just fail quickly, as it makes the code neater.

Reply:
👍